package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "@fastify/autoload",
-  "version": "5.8.0",
+  "version": "5.8.1",
   "description": "Require all plugins in a directory",
   "main": "index.js",
   "type": "commonjs",
   "types": "types/index.d.ts",
   "scripts": {
     "lint": "standard | snazzy",
     "lint:fix": "standard --fix | snazzy",
-    "test": "npm run unit && npm run typescript && npm run typescript:jest && npm run typescript:esm && npm run typescript:swc && npm run typescript:swc-node-register && npm run typescript:tsm && npm run typescript:tsx && npm run typescript:vitest && npm run typescript:esbuild",
-    "typescript": "tsd",
+    "test": "npm run typescript && npm run typescript:jest && npm run typescript:default && npm run typescript:swc && npm run typescript:swc-node-register && npm run typescript:tsm && npm run typescript:tsx && npm run typescript:vitest && npm run typescript:esbuild && npm run unit",
+    "typescript": "tap plugin add @tapjs/typescript && tsd",
```

Comment:
I'm really dissatisfied with launching this type of command, but I don't know how to do better with Tap 18.

Reply:
I'm considering exploring alternative approaches, I really don't like the "fix" I am doing. I am gonna delve deeper into their documentation and source code  and eventually open an issue on their repository. I could be missing something as I don't really know tap.

@mcollina @climba03003, do you have any suggestions for me?

Reply:
I don't think we should rely on `@tapjs/typescript`
It means when we are running under `tap`, it is trans-pile by `tap` instead of the loader.